README.md

```diff
@@ -129,6 +130,61 @@ This makes it easy to add custom data and telemetry within your function.
 
 IOpipe's functionality can be extended through plugins. Plugins hook into the agent lifecycle to allow you to perform additional analytics.
 
+### Profiler Plugin
+
+The IOpipe agent comes bundled with a profiler plugin that allows you to profile your functions with [cProfile](https://docs.python.org/3/library/profile.html#module-cProfile).
+
+Here's an example of how to use the profiler plugin:
+
+```python
+from iopipe import IOpipe
+from iopipe.contrib.profiler import ProfilerPlugin
+
+iopipe = IOpipe(plugins=[ProfilerPlugin()])
+
+@iopipe
+def handler(event, context):
+    # do something here
+```
+
+By default the plugin will be disabled and can be enabled at runtime by setting the `IOPIPE_ENABLE_PROFILER` environment variable to `true`/`True`.
```

Comment:
Let's move to the `IOPIPE_PROFILER*` namespace (`IOPIPE_PROFILER_ENABLE`)

Reply:
Changed to `IOPIPE_PROFILER_ENABLED` to be consistent with `IOPIPE_ENABLED`.